batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/controller/StudentListController.java

```diff
@@ -0,0 +1,187 @@
+package com.lftechnology.batch7crud.controller;
+
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Student;
+import com.lftechnology.batch7crud.service.StudentServices;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.Servlet;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.servlet.Servlet'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)